src/dispatchers/keyEventDispatcher.ts

```diff
@@ -0,0 +1,67 @@
+///<reference path="../reference.ts" />
+
+module Plottable {
+export module Dispatcher {
+  export class KeyEvent {
+    private static _DISPATCHER_KEY = "__Plottable_Dispatcher_Key";
+    private _connected = false;
+    private _keydownBroadcaster: Core.Broadcaster<Dispatcher.KeyEvent>;
+    // private _keyupBroadcaster: Core.Broadcaster<Dispatcher.KeyEvent>;
+    private _processDownCallback = (e: KeyboardEvent) => this._processKeydown(e);
```

Comment:
we can try to be a bit consistent with our instance var names.

Mouse has _mouseover _mousemove _mouseout

Maybe _keyDown ?